# HG changeset patch
# User Rik <rik@octave.org>
# Date 1437701504 25200
#      Thu Jul 23 18:31:44 2015 -0700
# Node ID 031e76c512424ba4ec7df29332ad5541bb53bb16
# Parent  421e3ebfca8d9422cbf5e7efc3387b3769bf9167
doc: Update polar() example in manual now that rticks calc is correct.

* plot.txi: Remove code from example which manually sets polar rtick limit.

diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -200,17 +200,16 @@ produces the figure shown in @ref{fig:er
 @DOCSTRING(loglogerr)
 
 Finally, the @code{polar} function allows you to easily plot data in
 polar coordinates.  However, the display coordinates remain rectangular
 and linear.  For example,
 
 @example
 polar (0:0.1:10*pi, 0:0.1:10*pi);
-set (gca, "rtick", [0:10:40]);
 @end example
 
 @noindent
 produces the spiral plot shown in @ref{fig:polar}.
 
 @float Figure,fig:polar
 @center @image{polar,4in}
 @caption{Polar plot.}
